src/ypspur_ros.cpp

```diff
@@ -507,6 +511,12 @@ class YpspurRosNode
               pnh_, param.name_, 1,
               boost::bind(&YpspurRosNode::cbDigitalOutput, this, _1, i));
         }
+        else if (digital_input_discrete_ && param.input_)
+        {
+          pubs_["di/" + param.name_] = compat::advertise<std_msgs::Bool>(
+              nh_, "di/" + param.name_,
+              pnh_, "di/" + param.name_, 1);
```

Comment:
New topics don't need to support old topic path
```suggestion
          pubs_["di/" + param.name_] = nh_.advertise<std_msgs::Bool>("di/" + param.name_, 1);
```